awx/api/views/__init__.py

```diff
@@ -2716,6 +2707,7 @@ class JobTemplateInstanceGroupsList(SubListAttachDetachAPIView):
     serializer_class = serializers.InstanceGroupSerializer
     parent_model = models.JobTemplate
     relationship = 'instance_groups'
+    filter_read_permission = False
```

Comment:
Are there any other items we would want to apply this to? i.e. Notifications, whatever else?

Reply:
Yes, there's a performance argument for using this, if possible. However, we do have a lot of views that are _reverse_ relationships. Even if a JT should list all of its credentials, that doesn't mean a credential should list all the JTs it is used in.

Reply:
I'm good with this as a start. We can add other pieces as needed.